scripts/database/README_upgrade_across_versions.txt

```diff
@@ -0,0 +1,64 @@
+We now offer an *EXPERIMENTAL* upgrade method allowing users to skip
+over a number of releases. E.g., it should be possible now to upgrade
+a Dataverse database from v4.8.6 directly to v4.10, without having to
+deploy the war files for the 5 releases between these 2 versions and
+to run the corresponding database upgrade scripts manually.
+
+One more time, it is *EXPERIMENTAL*! DO NOT attempt to run this script
+on your production database WITHOUT BACKING IT UP first!
+
+The script, upgrade.sh, must be run in this directory, as follows:
+
+./upgrade.sh [VERSION_1] [VERSION_2] [PG_HOST] [PG_PORT] [PG_DB] [PG_USER]
+
+Where 
+
+[VERSION_1] and [VERSION_2] are valid Dataverse release tags, for ex., v4.8.6 and v4.9.4;
+
+NOTE: it is your responsibility to make sure VERSION_1 is the actual
+version of your current database!
+
+[PG_HOST] is the server running PostgresQL used by your Dataverse
+[PG_PORT] the port on the PostgresQL server
+[PG_DB]   the name of the PostgresQL database used by your Dataverse
+[PG_USER] the name of the PostgresQL user used by your Dataverse
```

Comment:
From reading (vs testing), it looks like PG_USER does not need administrative access to the database (just to PG_DB).  If this is inaccurate, it might be worth clarifying.

Reply:
I say down below "just copy the values from your domain.xml and you'll be fine" - so it should probably be ok?